grails-app/domain/au/org/emii/portal/Server.groovy

```diff
@@ -223,4 +222,10 @@ class Server {
             operations << operation
         }
     }
+
+    def getUrlListDownloadSubstitutions() {
+        return grailsApplication.config.knownServers.find {
+            this.uri.startsWith(it.uri)
+        }?.urlListDownloadSubstitutions ?: [:]
```

Comment:
Good point about `find()` not necessarily finding anything. I was thinking about `findAll()` which returns an empty list if nothing is found.

Yep. Makes sense.